CONTRIBUTING.md

```diff
@@ -0,0 +1,83 @@
+# How to contribute
+
+Thanks for your interest in contributing to https://www.nysenate.gov/. We aim to
+provide transparency and accountability in New York State Senate. We welcome
+your help in making this site better.
+
+## Submitting changes
+
+First, please open a Github issue to discuss what you would like to change.
+[NYS Github Issues](https://github.com/nysenate/NYSenate.gov-Website-D9/issues)
+
+If you have a bug fix or new feature, you can fork the repository, make your
+changes, and submit a pull request.
+
+Please follow our coding conventions (below) and make sure all of your commits
+are atomic (one feature per commit).
+
+Always write a clear log message for your commits. Including the JIRA ticket
+number or Github issue number is helpful.
+
+`$ git commit -m "Issue-### A brief summary of the commit"`
```

Comment:
Need more direction on proper commit messages.  

- Imperative phrasing
- One-line summary, with optional explanations
- Reference to issue # is mandatory where applicable

Reference (or include direct links to) https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html and https://www.git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines